drivers/sx1280/include/sx1280_constants.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2022 Inria
+ * Copyright (C) 2020-2022 Université Grenoble Alpes
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_sx1280
+ * @{
+ *
+ * @file
+ * @brief       Internal addresses, registers and constants
+ *
+ * @author      Aymeric Brochier <aymeric.brochier@univ-grenoble-alpes.fr>
+ */
+
+#ifndef SX1280_CONSTANTS_H
+#define SX1280_CONSTANTS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief
+ *
+ * @see      https://github.com/lorabasics/lorabasicsmodem/blob/master/sx1280_driver/src/sx1280.c#L348
+ *
+ * sx1280_set_rx require this constant to be in continuous mode
+ */
+#define SX1280_RX_CONTINUOUS_MODE 0xFFFF
```

Comment:
It looks like there is an errata for this, maybe mention it if you do not want to implement the solution:
![image](https://user-images.githubusercontent.com/19396439/167562925-64254635-dc99-46ad-b1c2-0e459ee3c219.png)

Reply:
This has now been addressed.